devnet/substrate_alice.Dockerfile

```
@@ -0,0 +1,44 @@
+FROM parity/polkadot:v0.9.10 as polkadot
```

nit use `AS` so it's syntax highlighted better 😉 

```suggestion
FROM parity/polkadot:v0.9.10 AS polkadot
```